plugins/pip/pip.plugin.zsh

```diff
@@ -102,10 +108,25 @@ function pipupall {
   pip list --outdated --format freeze | cut -d= -f1 | ${=xargs} pip install --upgrade
 }
 
-# Uninstalled all installed packages
+# Uninstall all installed packages
 function pipunall {
   # non-GNU xargs does not support nor need `--no-run-if-empty`
   local xargs="xargs --no-run-if-empty"
   xargs --version 2>/dev/null | grep -q GNU || xargs="xargs"
   pip list --format freeze | cut -d= -f1 | ${=xargs} pip uninstall
 }
+
+# Install from GitHub repository
+function pipig {
+  pip install "git+https://github.com/$1/$2.git"
```

Comment:
I'm not sure that works for many projects.  Consider python/mypy, PyCQA/pylint, deepmin/dm-haiku, etc.  We really want to maximize the utility of this command so that we never have to type it in manually.

Reply:
Ah, okay yes that makes perfect sense.  I'll edit now.